test/src/app/with-scaled-background/page.tsx

```diff
@@ -1,94 +1,95 @@
 'use client';
 
-import { useState } from 'react';
-import clsx from 'clsx';
 import { Drawer } from 'vaul';
-import { DrawerDirection } from 'vaul/src/types';
 
-const CenteredContent = () => {
+export default function MyDrawer() {
```

Comment:
if possible I'd say to add an example of drawer that opens after a scroll and also the drawer to open another drawer, since these 2 situations are very common to real products and these bugs were spotted only in these circumstances

Reply:
Added a new page (/open-another-drawer), could you verify that it works for you please?

Reply:
running locally it works 👍 (can't verify from iphone atm)
![image](https://github.com/user-attachments/assets/35587e69-7353-4744-be96-e883a84e2ae8)